Dev-Corner/slides/tool_integration.html

```diff
@@ -977,11 +977,42 @@
                     <section data-markdown>
                         <script type="text/template">
                             <img src="../../shared/images/conda_logo.png" class="no-border"  height=25 />
+                            ### Quickstart
+                            #### Use Conda outside Planemo
+
+                            <div align="left">
+                            - Install a package
+                            </div>
+
+                            ```bash
+                            conda install bioconductor-deseq2
+                            ```
+
+                            <div align="left">
+                            - Install some packages within an isolate environment
+                            </div>
+
+                            ```bash
+                            $ conda create -n bioconductor-deseq2 bioconductor-deseq2 r-ggplot2
+                            $ conda env list
+                                bioconductor-deseq2   *  ~/miniconda2/envs/bioconductor-deseq2
```

Comment:
Maybe it would be clearer if you use a specific name for the env name (different from the one of the conda package), for example bioconductor-deseq2_env.